arviz/plots/essplot.py

```diff
@@ -26,12 +26,14 @@ def plot_ess(
     rug=False,
     rug_kind="diverging",
     n_points=20,
+    extra_methods=False,
     min_ess=400,
     ax=None,
     extra_kwargs=None,
+    text_kwargs=None,
     hline_kwargs=None,
     rug_kwargs=None,
-    **kwargs
+    **kwargs,
```

Comment:
Is it just the trailing comma that is adding this error?

Reply:
Now the error seems due to a missing trailing comma. Any idea on when an extra comma becomes a syntax error and when the syntax error is actually due to the extra comma missing? Are there any docs on this? Is it a black issue?

Reply:
https://github.com/python/black/issues/759

> Looks like we improperly merged two different syntax changes: `f(**x,)` (trailing comma in a call) first became legal in 3.5, but `def f(**x,): pass` (trailing comma in a definition) became legal only in 3.6. I'll try to disentangle the two in the code.

Reply:
I guess we have to wait until this fix is released then.